addons/mail/static/src/models/activity_button_view.js

```diff
@@ -1,11 +1,17 @@
 /** @odoo-module **/
 
+import { useRefToModel } from '@mail/component_hooks/use_ref_to_model';
 import { registerModel } from '@mail/model/model_core';
 import { attr, one } from '@mail/model/model_field';
 import { clear } from '@mail/model/model_field_command';
 
 registerModel({
     name: 'ActivityButtonView',
+    template: 'mail.ActivityButtonView',
+    templateGetter: 'activityButtonView',
+    componentSetup() {
+        useRefToModel({ fieldName: 'buttonRef', refName: 'button' });
+    },
```

Comment:
I'd rather keep the number of model definition sections to its bare minimum. With a few adjustments to our component hooks, I believe that this method could be moved among the model lifecycle hooks.

Reply:
I wouldn't do this, because `this` in this `componentSetup` is the one of component, not the model.
So it's best to keep isolated from other methods